recipes/opensim/bld.bat

```diff
@@ -0,0 +1,11 @@
+mkdir build
+cd build
+cmake -G "%CMAKE_GENERATOR%"^
+ -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"^
+ -DSIMBODY_HOME="%LIBRARY_PREFIX%"^
+ ..\OpenSim33-source
```

Comment:
The zip file uncompresses oddly and you have to enter this subdirectory. Also it includes a `__MACOSX__` directory that probably shouldn't be in the zip file.